### PR TITLE
chore: Update README.md

### PR DESCRIPTION
**Pull request by mikeselva123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,14 @@
 ### 🕶 Awesome Lists
 
 - [Abhishek Naiidu](https://github.com/abhisheknaiidu/awesome-github-profile-readme)
-
+- [A]
```
Comment (rishichitnis008) [inline README.md:11]: @mikeselva123 Please remove this

**Review by rishichitnis008 — CHANGES_REQUESTED** [review]
This looks awesome, but please remove the uneeded letter on line 11

File: README.md (hunk 1 above)
Comment (mikeselva123) [reply README.md]: Fixed it, Sorry about that.

**Review by rishichitnis008 — APPROVED** [review]
Awesome 🔥 🔥

**Pull request merged**